packages/sdk/src/util.test.ts

```diff
@@ -439,25 +454,123 @@ export async function createSpaceAndDefaultChannel(
     }
 }
 
+export async function createVersionedSpaceFromMembership(
+    client: Client,
+    spaceDapp: ISpaceDapp,
+    wallet: ethers.Wallet,
+    name: string,
+    membership: LegacyMembershipStruct | MembershipStruct,
+): Promise<ethers.ContractTransaction> {
+    if (useLegacySpaces() && isLegacyMembershipType(membership)) {
+        return await spaceDapp.createLegacySpace(
+            {
+                spaceName: `${name}-space`,
+                uri: `${name}-space-metadata`,
+                channelName: 'general',
+                membership,
+            },
+            wallet,
+        )
+    } else {
+        if (isLegacyMembershipType(membership)) {
+            // Convert legacy space params to current space params
+            membership = {
+                settings: membership.settings,
+                permissions: membership.permissions,
+                requirements: {
+                    everyone: true,
```

Comment:
should this be `everyone: await membership.requirements.everyone` ?

Reply:
no `await` - but yes, you're right.